packages/core/src/tools/tools.ts

```diff
@@ -97,7 +97,7 @@ export abstract class BaseTool<
    * @param description Description of what the tool does
    * @param isOutputMarkdown Whether the tool's output should be rendered as markdown
    * @param canUpdateOutput Whether the tool supports live (streaming) output
-   * @param parameterSchema JSON Schema defining the parameters
+   * @param parameterSchema Open API 3.0 Schema defining the parameters
```

Comment:
I think parameterSchema has a Schema type, which is a subset of openAPI v3.0.3 https://spec.openapis.org/oas/v3.0.3#schema. I am also open to leave it as it is.